docs/util.py

```diff
@@ -130,7 +192,54 @@ def plot_shaded_with_barbs(plot_ds,
     fig.colorbar(im, cax=cax)
 
 
+
```

Comment:
@rajveer43 can you remove this line as in standard practice only 2 empty lines between any function here I found 3 empty lines.